src/utils/assets.js

```diff
@@ -332,10 +332,17 @@ export const isMatchingCollectible = (
 export const mapWalletAssetsBalancesIntoAssetsByAddress = (
   walletAssetsBalances: WalletAssetsBalances,
   chainSupportedAssets: Asset[],
-): AssetByAddress => mapValues(
```

Comment:
Alternatively to two other comments in this file, consider wrapping `maValues` call with `omitNilProps` from `utils/object`